### PR TITLE
[MM-40038] Track crt events

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary
Add telemetry for key events on CRT

#### Ticket Link
[MM-40038](https://mattermost.atlassian.net/browse/MM-40038)

```release-note
Added telemetry on key events for CRT
```

### CODE REVIEW

File: components/post_view/post/post.jsx [hunk 1]
```diff
@@ -195,6 +196,7 @@ export default class Post extends React.PureComponent {
             (fromAutoResponder || !isSystemMessage) &&
             isEligibleForClick(e)
         ) {
+            trackEvent('crt', 'clicked_to_reply');
```
Comment (koox00) [inline components/post_view/post/post.jsx:199]: this is going to have a ton of false positives!

**Review by koox00 — APPROVED** [review]

File: components/post_view/post/post.jsx (hunk 1 above)
Comment (Willyfrog) [reply components/post_view/post/post.jsx:199]: you mean people missclicking? or something else?

**Review by AshishDhama — APPROVED** [review]

**Comment by cwarnermm** [comment]
@esethna - What updates, if any, are needed to the [Telemetry](https://docs.mattermost.com/manage/telemetry.html) product documentation page for this PR?

**Comment by esethna** [comment]
@Willyfrog on the above docs question ^

File: components/post_view/post/post.jsx (hunk 1 above)
Comment (koox00) [reply components/post_view/post/post.jsx:199]: sorry, I totally missed that, yep I meant people missclicking

**Pull request merged**